Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml

```diff
@@ -22,6 +22,57 @@ ParserParams:
 ParserQuery: |
```

Comment:
can you please add following property under ParserName?
EquivalentBuiltInParser: _ASim_Authentication_MicrosoftWindowsEvent

For reference: https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoASA.yaml